client/routing.js

```diff
@@ -33,8 +38,10 @@ const isProfitableToClaim = async (conf, txRequest, gasToClaim) => {
   const currentGasPrice = new BigNumber(await Util.getGasPrice(web3))
   const gasCostToClaim = currentGasPrice.times(gasToClaim)
 
-  if (gasCostToClaim.greaterThan(paymentWhenClaimed)) {
-    conf.logger.debug(`[${txRequest.address}] Not profitable to claim. gasCostToClaim: ${gasCostToClaim} | paymentWhenClaimed: ${paymentWhenClaimed}`)
+  const bountyProfitable = txRequest.bounty > (PROFITABILITY_INDEX * currentGasPrice)
```

Comment:
Isn't a `txRequest.bounty` a BigNumber? Please check whether you should use here `>` or `.greaterThan()`.